pkg/kubernetes/version.go

```diff
@@ -0,0 +1,97 @@
+package kubernetes
+
+import (
+	"fmt"
+	"sort"
+	"strconv"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+const (
+	// https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/
+	AnnotationSpinnakerArtifactVersion = `artifact.spinnaker.io/version`
+	AnnotationSpinnakerMonikerSequence = `moniker.spinnaker.io/sequence`
+)
+
+type SpinnakerVersion struct {
+	LongVersion  string
+	ShortVersion string
```

Comment:
Consider renaming these to `Long` and `Short` as `SpinnakerVersion.LongVersion` seems redundant.